core/storage/activity/gae_models.py

```diff
@@ -37,25 +43,33 @@ class ActivityReferencesModel(base_models.BaseModel):
     # in this list is a dict with two keys: 'type' and 'id'.
     activity_references = datastore_services.JsonProperty(repeated=True)
 
+    def __init__(self, *args, **kwargs): # pylint: disable=useless-super-delegation
+        # type: (*Any, **Any) -> None
+        super(ActivityReferencesModel, self).__init__(*args, **kwargs)
+
     @staticmethod
     def get_deletion_policy():
+        # type: () -> Any
```

Comment:
I think you can make these tighter. Let's try to avoid the use of Any everywhere.

Reply:
@seanlip We create enums in a very dynamic way using the function create_enum. Therefore mypy is unable to understand the type of our enums. Therefore we have to annotate our enums as type `Any`.
Already mentioned in the PR description. PTAL.

Reply:
I did read your description, but you have not provided any detail on alternative solutions.

This is one of those things that should be discussed in the doc. Using Any should really be the last resort and needs to be justified by "there is no other approach that would work".

Reply:
The only approach for this is to create enums like this:
```
class Color(Enum):
    RED = 1
    GREEN = 2
    BLUE = 3
```
I don't think this is feasible, right? @seanlip

Reply:
This needs to be discussed in your doc. When you make statements like "The only approach", those statements need justification.

Reply:
Added

Reply:
Btw, after we migrate to py3 I think we can introduce enums in a way that you mention above Hardik.

Reply:
Already changed ENUMs